### PR TITLE
Feature/testagent handles tls verified docker servers

### PR DESCRIPTION
**Pull request by kvahed**
* pass on environment to starters.
* mount certificate path into starters.

### CODE REVIEW

File: pkg/docker/docker_host.go [hunk 1]
```diff
@@ -44,17 +44,8 @@ func NewDockerHosts(endpoints []string, localHostIP, dockerIntf string) ([]*Dock
...
-	var err error
-	var client *dc.Client
-	if len(path) > 0 {
-		ca := fmt.Sprintf("%s/ca.pem", path)
-		cert := fmt.Sprintf("%s/cert.pem", path)
-		key := fmt.Sprintf("%s/key.pem", path)
-		client, err = dc.NewTLSClient(endpoint, cert, key, ca)
-	} else {
-		client, err = dc.NewClient(endpoint)
-	}
```
Comment (lamai93) [inline pkg/docker/docker_host.go]: What about this?

File: service/cluster/arangodb/machine.go [hunk 2]
```diff
@@ -321,19 +322,22 @@ func (c *arangodbCluster) createMachine(index int) (*arangodb, error) {
...
 		)
 	}
+	dockerCertPath := os.Getenv("DOCKER_CERT_PATH")
 	opts := dc.CreateContainerOptions{
 		Name: name,
 		Config: &dc.Config{
 			Image: c.ArangodbConfig.ArangodbImage,
 			Cmd:   args,
 			Tty:   true,
+			Env:   os.Environ(),
```
Comment (lamai93) [inline service/cluster/arangodb/machine.go]: This is a complete pass through of all envs from the client?

File: pkg/docker/docker_host.go (hunk 1 above)
Comment (kvahed) [reply pkg/docker/docker_host.go]: It's all covered, dude. That's the whole idea. It now handles both modes.

File: service/cluster/arangodb/machine.go (hunk 2 above)
Comment (kvahed) [reply service/cluster/arangodb/machine.go]: Yes. And specifically all DOCKER stuff.

**Pull request merged**